assets/js/app.js

```diff
@@ -1,3 +1,6 @@
+import '../css/app.css'
+import '../css/copy-button.css'
```

Comment:
Do we import CSS files inside a JS file? 🤔

Reply:
Another question: 
Is it better to add here just the
```js
import '../css/app.css'
```

And inside the `css/app.css` we add the:
```
@import ./copy-button.css
```
?

Reply:
The default webpack config provided by phoenix requires it, maybe we can add an issue to investigate if we can remove this later.